Runtime/Plugins/link.xml

```diff
@@ -0,0 +1,11 @@
+<linker>
+    <assembly fullname="MobiledgeX.MatchingEngineGrpc" preserve="all"/>
+
+    <!-- MobiledgeX.MatchingEngineGrpc Depedencies to preserve -->
+    <assembly fullname="Google.Protobuf" preserve="all"/>
+    <assembly fullname="Grpc.Core" preserve="all"/>
+    <assembly fullname="Grpc.Core.Api" preserve="all"/>
+    <assembly fullname="System.Buffers" preserve="all"/>
+    <assembly fullname="System.Memory" preserve="all"/>
+    <assembly fullname="System.Runtime.CompilerServices.Unsafe" preserve="all"/>
```

Comment:
Did you had to untick the allow unsafe code in Unity Project Settings

Reply:
Will test on device today.

Reply:
Build Target iOS
Using Editor 2020.2 and Build failed -> Linker error in XCode

On Editor LTS 2019.4.7, I got this error
I added link.xml in Assets folder
<img width="907" alt="Screen Shot 2021-02-25 at 10 53 10 PM" src="https://user-images.githubusercontent.com/24863504/109266009-4891be00-77bc-11eb-871f-b9aaa817a416.png">

What are the instruction to get it to work on iOS?